WordPressAuthenticator/Unified Auth/View Related/Get Started/GetStartedViewController.swift

```diff
@@ -752,7 +752,7 @@ private extension GetStartedViewController {
             return
         }
 
-        vc.loginFields = loginFields
+        vc.loginFields = loginFields.copy()
```

Comment:
I see that there are a few other places in the library where we're sending the reference to the same login fields instead of sending the copy, should we be worried about them? It makes sense just to handle the one with the problem at the moment.

Reply:
> I see that there are a few other places in the library where we're sending the reference to the same login fields instead of sending the copy, should we be worried about them?

Yes, we should probably be careful of this shared `LoginFields` behavior when navigating between screens and consider this behavior when looking into login issues. There are many mutations on `LoginFields` in various screens, but this PR is meant to just fix the specific crash/issue so I'll keep the copy just in this use case.